docs/user-guide/pod-preset/index.md

```diff
@@ -0,0 +1,540 @@
+spec:
+  selector:
+    matchLabels:
+      role: frontend
+  volumeMounts:
+    - mountPath: /etc/proxy/configs
+      name: proxy-volume
+  volumes:
+    - name: proxy-volume
+      emptyDir: {}
+```
+
+**Pod spec after admission controller:**
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: website
+  labels:
+    app: website
+    role: frontend
+  annotations:
+    podpreset.admission.kubernetes.io/allow-database: "resource version"
+    podpreset.admission.kubernetes.io/proxy: "resource version"
+spec:
+  containers:
+    - name: website
+      image: ecorp/website
+      volumeMounts:
+        - mountPath: /cache
+          name: cache-volume
+        - mountPath: /etc/proxy/configs
+          name: proxy-volume
+      ports:
+        - containerPort: 80
+      env:
+        - name: DB_PORT
+          value: 6379
+  volumes:
+    - name: cache-volume
+      emptyDir: {}
+    - name: proxy-volume
+      emptyDir: {}
+```
+
+### Conflict Example
+
+This is a example to show how a Pod spec is not modified by the Pod Injection
+Policy when there is a conflict.
+
+**User submitted pod spec:**
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: website
+  labels:
+    app: website
+    role: frontend
+spec:
+  containers:
+    - name: website
+      image: ecorp/website
+      volumeMounts:
+        - mountPath: /cache
+          name: cache-volume
+      ports:
+  volumes:
+    - name: cache-volume
+      emptyDir: {}
+        - containerPort: 80
+```
+
+**Example Pod Preset:**
+
+```yaml
+kind: PodPreset
+apiVersion: settings/v1alpha1
+metadata:
+  name: allow-database
+  namespace: myns
+spec:
+  selector:
+    matchLabels:
+      role: frontend
+  env:
+    - name: DB_PORT
+      value: 6379
+  volumeMounts:
+    - mountPath: /cache
+      name: other-volume
+  volumes:
+    - name: other-volume
+      emptyDir: {}
+```
+
+**Pod spec after admission controller will not change because of the conflict:**
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: website
+  labels:
+    app: website
+    role: frontend
+spec:
+  containers:
+    - name: website
+      image: ecorp/website
+      volumeMounts:
+        - mountPath: /cache
+          name: cache-volume
+      ports:
+  volumes:
+    - name: cache-volume
+      emptyDir: {}
+        - containerPort: 80
+```
+
+**If we run `kubectl describe...` we can see the event:**
+
+```
+$ kubectl describe ...
+....
+Events:
+  FirstSeen             LastSeen            Count   From                    SubobjectPath               Reason      Message
+  Tue, 07 Feb 2017 16:56:12 -0700   Tue, 07 Feb 2017 16:56:12 -0700 1   {podpreset.admission.kubernetes.io/allow-database }    conflict  Conflict on pod preset. Duplicate mountPath /cache.
+```
+
+## Deleting a Pod Preset
+
+Once you don't need a pod preset anymore, simply delete it with `kubectl`:
+
+```shell
+$ kubectl delete podpreset allow-database
+podpreset "allow-database" deleted
+```
+
+## Enabling Pod Preset
```

Comment:
Move this content directly after the "Creating a Pod Preset" header; the user needs to know this first.